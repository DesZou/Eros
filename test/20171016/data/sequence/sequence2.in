50 137495310 24
509924947 808578070 8351402 776095051 534585157 317916457 484548395 828990355 73238046 295128552 159073795 237455858 559474962 772863929 627244942 330369815 830283198 726144497 830186652 312318710 841409723 217307196 629244231 549988990 561412533 341845321 987747840 520201885 242771955 867074885 829245071 725730231 580508039 950450968 573594111 708994242 218164008 852359063 762615925 649519812 826242033 268887740 547197370 177027881 382581345 200046711 898082800 174627551 299791663 297572992 
50
26 28
46 49
38 41
43 49
7 23
29 32
2 48
35 45
44 44
14 29
33 43
19 47
5 44
36 49
28 30
16 22
31 42
46 49
34 40
40 46
1 1
20 34
23 49
7 11
10 50
31 46
19 48
25 32
9 46
7 47
3 5
45 47
35 48
22 32
13 23
14 20
21 23
4 28
15 32
49 49
23 45
28 36
44 49
38 45
23 26
32 33
38 39
36 46
35 50
13 31
