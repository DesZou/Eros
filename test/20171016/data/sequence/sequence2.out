0
0
0
0
0
0
290
0
0
0
0
21
148
0
0
0
0
0
0
0
0
0
10
0
171
0
28
0
119
168
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
